packages/docs/docs/getting-started/quickstart.mdx

```diff
@@ -67,7 +82,7 @@ animation, but for now there's not much to see.
 
 The scaffolding command will create several files for you, but for now we're
 going to focus on `src/scenes/example.tsx`, which is where we can add our
-animations. Open `example.tsx` in a text editor, and replace all code in the
+animations. Open `example.tsx` in a text editor (Ex] nano example.tsx), and replace all code in the
```

Comment:
Not sure it's worth being opinionated about which editor...

Reply:
It's just an example..............